src/pool/srv_pool.c

```diff
@@ -5381,6 +5381,9 @@ static int pool_find_all_targets_by_addr(struct pool_map *map,
  * \param[in]	svc		pool service
  * \param[in]	opc		update operation (e.g., POOL_EXCLUDE)
  * \param[in]	exclude_rank	for excluding ranks (rather than targets)
+ * \param[in]	extend_rank_list ranks list to be extended.
```

Comment:
[Not requesting changes] I think we should consider merging `extend_rank_list` with `tgts`/`tgt_addrs` going forward, as the set of parameters of this function is growing out of hand.

Reply:
yeah, indeed, I do not like current parameters list as well, a bit out control here. Though sharing the parameters might make it a bit complex.  I will check. Thanks.